content/blog/2023/2023-12-17-extension-functions-in-kotlin.md

```diff
@@ -0,0 +1,235 @@
+---
+title: "Exploring the Power of extension Functions in Kotlin"
+categories: ["Kotlin"]
+date: 2023-12-13 00:00:00 +1100
+authors: [ezra]
+excerpt: "In this tutorial, we'll discuss what extension functions are in kotlin and their usecases"
+image: images/stock/0104-on-off-1200x628-branded.jpg
+url: extension functions in kotlin
+---
+
+One of the Kotlin's standout features is extension functions, a mechanism that empowers developers to enhance existing classes without modifying their source code. In this blog post, we will embark on a journey to explore the fascinating realm of Kotlin extension functions, understanding their syntax, exploring use cases and recognizing their impact on code maintainability and readability.
```

Comment:
I took the liberty and dampened the language here a bit ... readers want to learn about extension functions, not "embark on a journey to explore the fascinating realm of Kotlin extension functions" ... :)